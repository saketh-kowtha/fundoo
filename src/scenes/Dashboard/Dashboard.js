import React from 'react'

import './Dashboard.scss'

import Header from './modules/Header/Header'
import Sidebar from './modules/Sidebar/Sidebar'
import Content from './modules/Content/Content'
import { connect } from 'react-redux';
import { withRouter } from "react-router-dom";

class Dashboard extends React.Component {

    state={
        sideBar: true
    }

    handleToggle = () => {
        this.setState({sideBar: !this.state.sideBar})
    }

    render() {
        return <div className="dashboard">
            <Header toggle={this.handleToggle} image={this.props.user.imageUrl} />
            <div style={{display: 'flex'}}>
                <Sidebar enable={this.state.sideBar}/>
                <Content />
            </div>
        </div>
    }
}


const mapToProps = (state) =>{
    return {
        user: state.user
    }
}

export default connect(mapToProps, null)(Dashboard)