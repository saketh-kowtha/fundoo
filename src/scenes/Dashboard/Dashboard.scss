@import "../../../index.scss";

.dashboard{
    height: 100vh;
    width: 100%;

    .border{
        border: 0.5px solid $borderColor;
    }
}


.dashboard > div{
    display: flex;
    width: 100%;
}