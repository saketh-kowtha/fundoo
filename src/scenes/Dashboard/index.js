import React from 'react'

class Dashboard extends React.Component {
    render() {
        return <div>Welcome To Dahboard</div>
    }
}


export default Dashboard