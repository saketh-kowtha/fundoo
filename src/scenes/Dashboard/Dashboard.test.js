import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from './Dashboard';

describe('Dashboard', () => {
  
    it('should render correctly ', () => {
        const component = shallow(<Dashboard />);
        expect(component).toMatchSnapshot();
    });
    
    it('should HashRouter have noslash prop ', () => {
        const component = shallow(<Dashboard />);
        expect(component).toMatchSnapshot();
    });
    
    

});