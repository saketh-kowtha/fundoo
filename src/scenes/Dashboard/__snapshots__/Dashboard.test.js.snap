// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should HashRouter have noslash prop  1`] = `ShallowWrapper {}`;

exports[`Dashboard should render correctly  1`] = `ShallowWrapper {}`;
